zebrad/src/application.rs

```diff
@@ -111,18 +111,13 @@ impl Application for ZebradApp {
             .as_ref()
             .expect("config is loaded before register_components");
 
+        let default_filter = if command.verbose { "debug" } else { "info" };
```

Comment:
Have you tested if the non-server commands output any info-level logs?
If they do, using info as a default will break revhex and generate output, and be annoying for help, version, and no arguments.

Let's make sure that the revhex output is exact, and the output of the other commands starts with the expected first line.

Reply:
Thanks, i just checked that. No info level log is printed in `generate`, `help`, `revhex` and `version` commands.

Reply:
I opened #845 and #847 so we test for accidental logs in command output.